jakartaee9/JakartaEE9ReleasePlanFAQ.md

```diff
@@ -35,3 +39,8 @@ Pruned Jakarta EE Projects will leave their property names in the `javax.*` name
 
 Yes, the proposed target namespace is `https://jakarta.ee/xml/ns/jakartaee/`.  
 This work is being tracked [via this bug](https://github.com/jakartaee/jakarta.ee/issues/592).
+
+### Keep the old `javax` signature files?
+
+There is no reason to keep these old `javax` signature files since only the `jakarta` signatures will be developed, maintained, and tested.
+To clarify, this question is referring to the `javax` signature files in [this directory](https://github.com/eclipse-ee4j/jakartaee-tck/tree/master/src/com/sun/ts/tests/signaturetest/signature-repository).
```

Comment:
I still think you could clarify that only the most current signature files need to be kept;
whether the old ones are javax or jakarta doesn't matter.

Reply:
Thanks, Bill.  Since this FAQ is for the Jakarta EE 9 release plan and `jakarta` signature files do not exist prior to Jakarta EE 9, I think I'll just stick with the `javax` reference.